src/server/svr_func.c

```diff
@@ -960,6 +965,73 @@ unset_node_fail_requeue(void)
 		  LOG_NOTICE, msg_daemonname, log_buffer);
 }
 
+/*
+ *
+ * @brief
+ *	Set resend_term_delay attribute.
+ *
+ * @par Functionality:
+ *	This function sets the resend_term_delay server attribute.
+ *	resend_term_delay can not be < 0 and > 1800.
+ *
+ * @param[in]	pattr	-	ptr to attribute
+ * @param[in]	pobject	-	pointer to some parent object.(required but unused here)
+ * @param[in]	actmode	-	the action to take (e.g. ATR_ACTION_ALTER)
+ *
+ * @return	int
+ * @retval	PBSE_NONE
+ *
+ */
+int
+set_resend_term_delay(attribute *pattr, void *pobject, int actmode)
+{
+	if (actmode == ATR_ACTION_FREE)
+		return (PBSE_NONE);
+
+	if ((actmode == ATR_ACTION_ALTER) ||
+	    (actmode == ATR_ACTION_RECOV)) {
+
+		if (pattr->at_val.at_long >= 0 && pattr->at_val.at_long <= 1800) {
+			resend_term_delay = pattr->at_val.at_long;
+		} else {
+			return (PBSE_BADATVAL);
+		}
+		sprintf(log_buffer,
+			"resend_term_delay value changed to %ld",
+			resend_term_delay);
+		log_event(PBSEVENT_ADMIN, PBS_EVENTCLASS_SERVER,
+			  LOG_NOTICE, msg_daemonname, log_buffer);
+	}
+
+	return (PBSE_NONE);
+}
+
+/*
+ *
+ * @brief
+ *	Unset resend_term_delay attribute.
+ *
+ * @par Functionality:
+ *	This function unsets the resend_term_delay server attribute
+ *	by reverting it back to it's default value.
+ *
+ * @param[in]	void
+ *
+ * @return	void
+ *
+ */
+void
+unset_resend_term_delay(void)
+{
+	resend_term_delay = PBS_RESEND_TERM_DELAY_DEFAULT;
+
+	sprintf(log_buffer,
+		"resend_term_delay reverting back to default val %ld",
```

Comment:
I can't remember if log_eventf() made it into open PBS.  At some point I wrote a function which combines the sprintf()/log_event() calls into one.  If that is in OpenPBS, I'd use it.  If not, please ignore me :)

Reply:
Yes, `log_eventf()` is in OpenPBS. I changed it. Thx.